src/server/services/procedures/chart/chart.js

```diff
@@ -207,6 +208,22 @@ chart.draw = function(lines, options={}){
         };
     }
 
+    if (options.logscale) {
+        const val = options.logscale;
+        if (typeof val === 'string') opts.logscale = { axes: val, base: 10 };
+        else if (Array.isArray(val)) {
+            const axes = val[0];
+            let base = val[1];
+            if (typeof axes !== 'string') throw Error('logscale axes was not a string (text)');
```

Comment:
From the gnuplot reference, it can actually be "any combination of `x`, `x2`, `y`, `y2`, `z`, and `cb` in any order". But I could add some regex to check that.

Reply:
Oh, and for the if statement for string type, the only time I've seen that be invoked is if you pass it a list.

Reply:
Do you have an example where we might use some of the other options? If they aren't pressing for us, I would opt for supporting just "x" and "y". This should make the error messages easier to understand, too.